src/box/xrow_update.c

```diff
@@ -169,13 +169,8 @@ xrow_update_read_ops(struct xrow_update *update, const char *expr,
 
```

Comment:
In the second commit you updated test-run submodule. I suppose it was an accident?

Reply:
Fixed.